### PR TITLE
Solution box branch

### PR DESCRIPTION
**Pull request by DanielLai20**
Added solutions box to code editor form as well as solutions variables

### CODE REVIEW

**Comment by marcgabe15** [comment]
please fix merge conflicts (i.e nothing should be in src so feel free to delete conflicting files I'm pretty sure?)

File: src/pages/ProblemEditor/ProblemEditorContainer/problemEditorContainerSlice.spec.ts [hunk 1]
```diff
@@ -0,0 +1,288 @@
+import problemEditorContainerSlice, {
```
Comment (marcgabe15) [inline src/pages/ProblemEditor/ProblemEditorContainer/problemEditorContainerSlice.spec.ts:1]: Question: was this copy pasted from the past src folder or are these new tests?

File: components/ProblemEditor/CodeEditorForm/CodeEditorForm.tsx [hunk 2]
```diff
@@ -202,6 +210,27 @@ export const CodeEditorForm = ({ formRef }: Props) => {
                 />
               </Paper>
             </Box>
+            <Box flexGrow={1} display="flex" flexDirection="column">
```
Comment (marcgabe15) [inline components/ProblemEditor/CodeEditorForm/CodeEditorForm.tsx:213]: This creates the UI, and I assume there will be a subsequent PR on the backend to store the solution?

**Review by marcgabe15 — APPROVED** [review]
Looks good! Just have a question about the test.spec file, also right now this PR creates a UI to have a solution box but how will you send the solution to the back end and store it?

File: components/ProblemEditor/CodeEditorForm/CodeEditorForm.tsx (hunk 2 above)
Comment (DanielLai20) [reply components/ProblemEditor/CodeEditorForm/CodeEditorForm.tsx:213]: yes, this was just the frontend part, Benny and I are currently working on the backend portion of this feature

File: src/pages/ProblemEditor/ProblemEditorContainer/problemEditorContainerSlice.spec.ts (hunk 1 above)
Comment (DanielLai20) [reply src/pages/ProblemEditor/ProblemEditorContainer/problemEditorContainerSlice.spec.ts:1]: i meant to delete it and commit it, will do it now.

**Review by marcgabe15 — APPROVED** [review]

**Review by kapooramanpreet — APPROVED** [review]
LG!

**Review by kapooramanpreet — APPROVED** [review]
LG!

**Pull request merged**